crowdkit/aggregation/classification/glad.py

```diff
@@ -61,14 +63,14 @@ class GLAD(BaseClassificationAggregator):
 
 
     Args:
-        max_iter: Maximum number of EM iterations.
-        eps: Threshold for convergence criterion.
-        silent: If false, show progress bar.
-        labels_priors: Prior label probabilities.
-        alphas_priors_mean: Prior mean value of alpha parameters.
-        betas_priors_mean: Prior mean value of beta parameters.
-        m_step_max_iter: Maximum number of iterations of conjugate gradient method in M-step.
-        m_step_tol: Tol parameter of conjugate gradient method in M-step.
+        n_iter: The number of EM iterations.
```

Comment:
No, it's the maximum number of iterations

Reply:
fixed